salsah/src/public/js/jquery.extsearch.js

```diff
@@ -500,7 +500,7 @@
 								for (var i in data.vocabularies)
 								{
 									if (data.vocabularies[i].active) {
-										vocsel.append(tmpele = $('<option>', {value: data.vocabularies[i].id}).append(data.vocabularies[i].longname + ' [' + data.vocabularies[i].shortname + ']'));
+										vocsel.append(tmpele = $('<option>', {value: data.vocabularies[i].id}).append(data.vocabularies[i].longname + ' [' + data.vocabularies[i].id.substr(data.vocabularies[i].id.lastIndexOf('/') + 1) + ']'));
```

Comment:
Could you write a utility function for this and call it several times? Please add a comment to it that explains what it does.

If I understand correctly, you are getting the project shortcode from the IRI.

Reply:
But the shortname still exists:

```
"vocabularies":[  
      {  
         "shortname":"beol",
         "description":"Bernoulli-Euler Online",
         "uri":"http://www.knora.org/ontology/beol",
         "id":"http://www.knora.org/ontology/beol",
         "project_id":"http://rdfh.ch/projects/yTerZGyxjZVqFMNNKXCDPF",
         "longname":"Bernoulli-Euler Online",
         "active":true
      }
...
```

Reply:
@subotic Is this correct?

Reply:
Yes, seems correct, since there is no `shortcode` defined for the `beol` project.